### ISSUE TITLE
Combobox and DatePicker are closing unexpectedly (IE11 / Edge)

### ISSUE THREAD

**Issue by rstabenow** [telerik/kendo-angular#1150]
## I'm submitting a...
* Bug report

## Current behavior
In some cases (most probably related to the attached EventHandlers to the window.scroll event) popups of input components like the Combobox and Datepicker are closing instantly or when scrolled.

## Expected behavior
* Combobox should stay open, when the user scrolls through its items.
* Datepicker should stay open for the user to select a date

## Minimal reproduction of the problem with instructions
The issue can't be reproduced in Plunker, I attached a demo working in Edge:
[test-kendo-popups.zip](https://github.com/telerik/kendo-angular/files/1551603/test-kendo-popups.zip)

* attach handler with EventListenerOptions to "scroll" event of window object (see main.ts)
`Observable.fromEvent(window, 'scroll', { }).subscribe(e => console.log(e));`

* try to open a Datepicker or scroll an opened Combobox in Edge

If any handler without EventListenerOptions is attached before, the issue will not occur. You can test this, by uncommenting line 8 in main.ts.

## What is the motivation or use case for changing the behavior?
The issue prevents from delivering good user experience in Edge/IE.

## Environment

Package versions:
package.json is attached

Browser:
- IE version 11
- Edge version 15

System:
- TypeScript version: 2.5.3 and 2.6.2
- Node version: 7.9.0
- NPM version: 4.2.0
- Platform:  Windows

The issue appeared in a rather complex scenario with a dynamic form component, which has to be compiled at runtime. The first instance of the dynamic form worked just fine, but if it was destroyed and a second one was created, the described problem appeared.
I debugged the application and found out, that the popups were closed by a scroll event inside the component, which bubbled to the window. I tried to log the scroll events of the window and to my surprise  the issue disappeared completely.
After that I tried to handle the event in passive mode to investigate further, which made the problem reproducable in the first instance or even without the dynamic component.

**Comment by ggkrustev** [comment]
Hi @rstabenow, thanks for the report.

Indeed, the popup immediately closes in IE11 when `window` scroll event is attached using `Observable`. After further investigation, we noticed that the `scroll` stream is initiated by `kendo-virtualization` scroll, which is incorrect in terms of how scroll event bubbling works. Basically, the scroll event shouldn't be bubbled when an inner element is scrolled. This leads me to a conclusion that there is something incorrect with the `Observable.fromEvent` method in this case.
Using a `fromEventPattern` works just fine even in IE11. I would suggest you try this approach instead.

```
import { Observable } from 'rxjs/Observable';
import 'rxjs/add/observable/fromEventPattern';

const scrolls = Observable.fromEventPattern(
    handler => window.addEventListener('scroll', handler),
    handler => window.removeEventListener('scroll', handler)
);

scrolls.subscribe(x => console.log(x));
```

**Comment by ggkrustev** [comment]
The issue has been addressed in @progress/kendo-angular-popup@1.6.0 or later.
